content/departments/engineering/dev/tools/scaletesting.md

```diff
@@ -119,6 +126,62 @@ or to start it:
+
+The instance can easily be started/shut down with the following commands:
+
+- Start up: `gcloud compute instances start ghe-scaletesting --zone=us-central1-f --project sourcegraph-scaletesting`
+- Shut down: `gcloud compute instances stop ghe-scaletesting --zone=us-central1-f --project sourcegraph-scaletesting`
+
+When starting the instance, it could take up to 5 mins for the application to set up before fully operational. The UI will indicate when it is ready to be used.
+
+### Scaling the instance
+
+The instance has been set up to handle 10k users based off of the [recommended specs](https://docs.github.com/en/enterprise-server@3.6/admin/installation/setting-up-a-github-enterprise-server-instance/installing-github-enterprise-server-on-google-cloud-platform#minimum-requirements) from GitHub.
+
+It can also easily be scaled up or down with the following steps:
+
+**Note:** Be cautious when scaling up, as we can only scale up and down CPU/Mem but not disk. To avoid overprovisioning, only increase disk size when needed.
+
+#### Increase/Decrease CPU and Memory
+
+1. Ensure the instance has been stopped by going into the GCP Console or running the shutdown command
+   `gcloud compute instances stop ghe-scaletesting --zone=us-central1-f --project sourcegraph-scaletesting`.
+2. Edit the machine type - We recommended staying within the `n2-highmem-xx` family as it's configuration best suits GitHub's requirements
+   ![edit-machine-type](https://storage.googleapis.com/sourcegraph-assets/handbook/edit-machine-type.png)
+3. Click Save, and start the instance back up.
+
+#### Increase Disk
+
+**Note:** Downsizing disk is not supported at the moment, please proceed with caution before increasing disk size.
+
+Increasing disk is a 2 part process - you have to increase disk on the VM, and then expand the ghe filesystem for the changes to be applied.
+
+1. Ensure the instance has been stopped by going into the GCP Console or running the shutdown command
+   `gcloud compute instances stop ghe-scaletesting --zone=us-central1-f --project sourcegraph-scaletesting`.
+2. In the Google Cloud Console, go to the Disks page under Compute Engine
+3. Identify which disk you want to increase
+   ![select-disk](https://storage.googleapis.com/sourcegraph-assets/handbook/increase-disk.png)
+4. Click Edit and enter the new size for your disk.
+5. Click Save, and start the instance back up.
+6. Follow the [instructions](https://docs.github.com/en/enterprise-server@3.6/admin/enterprise-management/updating-the-virtual-machine-and-physical-resources/increasing-storage-capacity#increasing-the-data-partition-size) from GitHub on how to expand the filesystem.
+
+### Logging in
+
+The instance has been set up with an admin account, please find the credentials in our [1Pass vault](https://start.1password.com/open/i?a=HEDEDSLHPBFGRBTKAKJWE23XX4&v=cjfb3n4rqj6s7mu3dkbci4dk2u&i=q3g4ywrebjiqmti2xro5tq5jwa&h=team-sourcegraph.1password.com)
+
+### Licenses
+
+TBD
```

Comment:
Do we want to attract more visibility to our unusual license situation? This block could just be a "Reach out to DevEx when experiencing issues related to licensing" or something. If we think it's fine to be more informative then go for it

Reply:
Let's drop this section for now @kalanchan